src/adapters/mc/asb/AzureLinuxBaseline_DeployIfNotExists.json

```diff
@@ -291,6 +312,10 @@
               {
                 "field": "Microsoft.GuestConfiguration/guestConfigurationAssignments/complianceStatus",
                 "equals": "Compliant"
+              },
+              {
+                "field": "Microsoft.GuestConfiguration/guestConfigurationAssignments/parameterHash",
```

Comment:
Whats the point of this parameter, how is it used?

Reply:
Please see the PR description. This is the policy parameter that allows an admin configure the logging level for Azure OSConfig on the target machine

Reply:
This is how such policy parameters need to be specified in policy definition, look at the SSH polkicy def for another example